docs/docs/how-to/performance/resolving-out-of-memory-issues.md

```diff
@@ -48,7 +48,7 @@ This will output every second profile information like the following:
 
 **Second, you can use an experimental tool** created by the Gatsby team **to generate build memory charts**. [This tool is called gjl](https://github.com/pieh/gjl) and is a wrapper around the gatsby process. Instructions for setting `gjl` up are in the README. A sample graph is below:
 
-![](https://lh6.googleusercontent.com/j9tUf6rJuyIgFP_-XJ7KvqGZrmA8BzRj43hoSgIxC5m8LMTcAlmM5BOO5KF6Cw3VKzkHexzOwK3UFpEItp4KHq7CTUvG09l1hqu6zLeMYPuBQoy0YD9ruPssxsG7BAmsYHeKcYhB)
+![A chart showing the Gatsby develop process. The left Y-Axis shows the event loop delay, the right Y-Axis the memory usage. The X-Axis is the time in seconds. The memory usage and event loop delay are plotted as a line chart over the time, in the background the different stages of the build are shown.](https://lh6.googleusercontent.com/j9tUf6rJuyIgFP_-XJ7KvqGZrmA8BzRj43hoSgIxC5m8LMTcAlmM5BOO5KF6Cw3VKzkHexzOwK3UFpEItp4KHq7CTUvG09l1hqu6zLeMYPuBQoy0YD9ruPssxsG7BAmsYHeKcYhB)
```

Comment:
@calcsam  Could we copy this image locally so that it can be processed with Sharp. This is embedding the full 680Kb png with no resizing.